content/surveys/_index.en.md

```diff
@@ -5,35 +5,94 @@ tags: ["wiki"]
 
-A design effect of 1.407 suggests that the variability (variance or the square
-of the standard error) of the estimate under the chosen design is 40.7% larger
-than that of the same-sized simple random sampling. Similarly, a design effect
-of 2.358 suggests that the variability of the estimate allowing for clustering,
-stratification and sampling weights is 135.8% larger than that would come from
-the same sample size if simple random sampling were used.
+# Getting started with Survey templates in R?
 
-Rearranging the formula: `deff=1+(n-1) x rho` ==> `rho=(deff-1)/(n-1)` where
-`deff=2.358` and `n=20`, you can get intraclass correlation coefficient
-`(rho)=0.071`.
+This section (INSERT LINK TO THE WALK THROUGH ON SURVEY DATA ANALYSIS) offers you a step by step guide on how to use the survey templates to fit your data and run the analysis.
+
+# Basic concepts of survey data analysis
+
+This website will not cover the detailed aspects of designing, implementing and analysing a survey in MSF contexts. There are many other training and briefing materials that address this aspect.
+
+We will cover a few crucial concepts which are required for the understanding and use of the templates for your survey analysis.
+
+## Sampling strategy
+There are different ways to sample populations for surveys. The most commonly used in MSF settings are:
+
+1. **Simple random sampling (SRS)**: requires a comprehensive sampling frame (i.e. total list of households inside a refugee camp or 
+GPS based sampling in a known area).
+2. **Cluster based sampling**: most commonly used in combination with sampling villages proportional to population size.
+3. **Stratified samples**: uses either SRS or cluster based sampling but in two different adminstrative areas in order to obtain
+precise prevalence estimates for them.
+
+The templates can accomodate analysis of data from all sampling designs.
+
+## Sample size
+When calculating a sample size for your survey you take the below parameters into consideration:
+
+* **Estimate of prevalence**
+* **Design effect or intraclass coefficient**
+* **Precision required around your estimate of prevalence**
+
+You might consider recalculating these parameters once the survey is completed, in order to check that your original assumptions were accurate.
+
+### Estimate of prevalence
+You will need to use an estimate of the prevalence/coverage of the your main outcome of interest in your to-be-surveyed population (prevalence of malnutrition, measles vaccination coverage etc.). You will use published or grey literature to determine a logical estimate of this parameter.
+
+Note: the largest sample size you will need will be that which is calculated using an estimate of prevalence of 50%. The closer to 0% or 100% prevalence you set your estimate, the smaller your sample size will be. Therefore, if you have no idea what your estimate of prevalence is, try and use 50% and the largets possible sample size.
```

Comment:
For this template, it's possible to display notes like this in separate boxes to help break up the wall of text: https://learn.netlify.com/en/shortcodes/notice/

```suggestion
{{% notice note %}}
The largest sample size you will need will be that which is calculated using an estimate of prevalence of 50%. The closer to 0% or 100% prevalence you set your estimate, the smaller your sample size will be. Therefore, if you have no idea what your estimate of prevalence is, try and use 50% and the largest possible sample size.
{{% /notice %}}
```